### ISSUE TITLE
How do i close mask branch？

### ISSUE THREAD

**Issue by DenceChen** [matterport/Mask_RCNN#1369]
i want just finetune bbox, and do not have segmention

**Comment by itachi1232gg** [comment]
why don't you use faster-rcnn?

**Comment by DenceChen** [comment]
@itachi1232gg 老哥有好的frcnn框架推荐么？keras的

**Comment by itachi1232gg** [comment]
@DenceChen 你git上找找吧， 应该挺多的， 我之前用的是pytorch的。

**Comment by ApoorvaSuresh** [comment]
Hi, 
did you find a way to do this?

**Comment by DenceChen** [comment]
@Apsu95 you just need set the mask-weight equal to 0 when you training
